### DISCUSSION TITLE
Support for likes/rating scrobbling to listenbrainz/last.fm

### DISCUSSION THREAD

**Discussion by ak1932** [navidrome/navidrome#3169] [Ideas]
Right now only the listens are scrobbled to the services enabled.
Likes and ratings are also very useful for developing a good model of the taste of your music on listenbrainz and last.fm.
For listenbrainz atleast, it seems to be a trivial addition.

I apologize if this is already implemented and I am just too dumb to find it :)

Finally thanks for this incredible piece of software

**Comment by kgarner7** [comment]
There is a PR for this (https://github.com/navidrome/navidrome/pull/2058), but likely on the backburner until https://github.com/navidrome/navidrome/pull/2709 is ready
